api/docker-compose.yml

```
@@ -0,0 +1,7 @@
+version: '2'
+services: 
+    reddis:
```

Should this be just `redis`?
```suggestion
    redis:
```